### DISCUSSION TITLE
Failed scores have incorrect max combo and total score

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#19251] [Ideas] [answered]
as stated in the title currently F scores calculate their 1 mill score seemingly based on the objects played uptil failure, which  gives nonsense scores like this 
![osu_2022-07-20_18-52-26](https://user-images.githubusercontent.com/74463310/180040786-6d59549c-89d8-44f0-8740-46dbe00bad3e.jpg)
combo is also broken for the same reasons

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/19259
